### DISCUSSION TITLE
Crop conditioning for SDXL

### DISCUSSION THREAD

**Discussion by mar-muel** [huggingface/diffusers#6674] [Q&A]
Hello all - I'm playing around with crop conditioning for SDXL using the example code as per docs

```python
from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_pretrained(
      "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, variant="fp16", use_safetensors=True
  ).to("cuda")

prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
for i in range(16):
    print(i)
    image = pipeline(prompt=prompt, crops_coords_top_left=(256, 0), num_inference_steps=30).images[0]
    image.save(f'test{i}.png')
 ```

However, I can't reproduce the effect and the generated images (see below) show the character still very much in the center.

Is this expected for SDXL and crop conditioning just doesn't work so well in practice?

![test0](https://github.com/huggingface/diffusers/assets/19345805/7efbd0e6-ea1c-4b8a-95ad-8e2555439c83)
![test1](https://github.com/huggingface/diffusers/assets/19345805/4b18a16f-b88c-4c30-bf4a-2f0cd975760e)
![test2](https://github.com/huggingface/diffusers/assets/19345805/0999adbe-ef28-4dfd-8e0f-72e7f2baadbd)
![test3](https://github.com/huggingface/diffusers/assets/19345805/7de9c8fb-c18e-4cc7-8d4c-26a455556e06)
![test4](https://github.com/huggingface/diffusers/assets/19345805/c7ed59db-0f53-404c-be04-729614fad284)
![test5](https://github.com/huggingface/diffusers/assets/19345805/95b73c60-3727-4945-ac6a-55f379f6a45e)
![test6](https://github.com/huggingface/diffusers/assets/19345805/30a85688-ca84-475b-8cf7-48fc82f3ad76)

**Comment by sayakpaul** [comment]
Cc: @stevhliu

**Comment by sayakpaul** [comment]
<table>
    <tr>
        <td><img src="https://github.com/huggingface/diffusers/assets/22957388/b139a380-fedd-4b0a-93eb-2fd21d6f6289" alt="Description of Image 1"></td>
        <td><img src="https://github.com/huggingface/diffusers/assets/22957388/2aa1b79f-35a7-445c-848d-e01df180bc60" alt="Description of Image 2"></td>
        <td><img src="https://github.com/huggingface/diffusers/assets/22957388/2e96334b-5789-49db-9151-8060a849c083" alt="Description of Image 3"></td>
        <td><img src="https://github.com/huggingface/diffusers/assets/22957388/59fdb789-19a0-4ce2-bdf9-1366ec38541e" alt="Description of Image 4"></td>
    </tr>
</table>


Running for more steps seems to be a little bit better. 

```python
from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_pretrained(
      "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, variant="fp16", use_safetensors=True
  ).to("cuda")

prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
for i in range(4):
    print(i)
    image = pipeline(prompt=prompt, crops_coords_top_left=(256, 0)).images[0]
    image.save(f'test{i}.png')
```
